### DISCUSSION TITLE
How can the ansible-lint warning colour be controlled?

### DISCUSSION THREAD

**Discussion by chriscroome** [ansible/ansible-lint#4461] [Q&A]
I have a terminal set to white on black, `ansible-lint` warnings are black on black, I have this in `ansible.cfg` to fix an issue like this with Ansible itself:

```ini
[colors]
debug = blue
```

But I can't work out how to set the `ansible-lint` warning colour, can someone explain how this can be done?

**Comment by sdarwin** [comment]
This is surprising!!  Also getting that.

> I have a terminal set to white on black, ansible-lint warnings are black on black.

**Comment by chriscroome** [comment]
See the follow to this issue here https://github.com/ansible/ansible-lint/issues/4469
